resmo/README.md

```diff
@@ -0,0 +1,61 @@
+## Overview
+[Resmo][1] is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
+Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
+
+What does Resmo offer to Datadog users?
+- Collect and monitor all your Datadog assets on a single platform.
+- Query your Datadog monitors, roles, permissions, API Keys, and more.
+- Set up rules to continuously assess your Datadog resources.
+- Set up and receive rule notifications based on your Datadog resource configurations.
+
+
+## Setup
+
+1. Go to https://resmo.com, log in and navigate to **Integrations**.<br/>
+
+![Resmo Screenshot][2]
+
+2. Click **Add Integration** and select **Datadog**. Write a descriptive name and optionally a description.<br />
+
+![Resmo Screenshot][3]
+
+3. Go to your Datadog account and create a new organization [API key][12] and a user [Application key][13].<br />
+
+4. Return to your Resmo account and enter the copied keys into the Organization API Key and User App Key fields. <br />
+
+![Resmo Screenshot][7]
+
+5. Select a site from the dropdown menu on the last field.
+
+![Resmo Screenshot][8]
+
+6. Click the **create** button, and you're ready to query your Datadog resources.
+
```

Comment:
```suggestion
## Data Collected

See [Resmo Datadog Resources][9] for the list of resources collected from your Datadog account.

```

Reply:
Optional suggestion to move the **Data Collected** section above **Uninstallation** -- I think that users will want to see the value of the integration after it's installed, and then learn how to uninstall if needed -- but that's just my opinion, so feel free to ignore this suggestion if this format is intentional for some other reason.